### PR TITLE
add warning for non conforming TODO's

### PR DESCRIPTION
**Pull request by erwinmombay**
requires the `TODO(username, #prid)` format

### CODE REVIEW

File: build-system/eslint-rules/todo-format.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+module.exports = function(context) {
+  return {
+    Program: function(node) {
+      if (node.comments) {
+        node.comments.forEach(function(comment) {
+          if (/TODO/.test(comment.value) &&
+                !/TODO\(\w+,\s*#\d{1,}\)/.test(comment.value)) {
+            context.report(comment,
+                'TODO\'s must be in TODO(username, #1234) format. Found: ' +
```
Comment (jridgewell) [inline build-system/eslint-rules/todo-format.js]: `@username`

Comment (rsimha-amp) [inline build-system/eslint-rules/todo-format.js]: Nit: Remove apostrophe from "TODO's". (See https://english.stackexchange.com/a/56010)

Comment (erwinmombay) [inline build-system/eslint-rules/todo-format.js]: done.

File: build-system/eslint-rules/todo-format.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+module.exports = function(context) {
+  return {
+    Program: function(node) {
+      if (node.comments) {
+        node.comments.forEach(function(comment) {
+          if (/TODO/.test(comment.value) &&
+                !/TODO\(@?\w+,\s*#\d{1,}\)/.test(comment.value)) {
+            context.report(comment,
+                'TODOs must be in TODO(username, #1234) format. Found: ' +
```
Comment (jridgewell) [inline build-system/eslint-rules/todo-format.js]: There's no `@`. 😝

Comment (erwinmombay) [inline build-system/eslint-rules/todo-format.js]: just pushed :D

**Comment by dvoytenko** [comment]
@erwinmombay Looks good. So, will these be for now non-travis-killing? It looks like travis fails at this time.

**Comment by erwinmombay** [comment]
@dvoytenko fixed the bug

**Pull request merged**